### ISSUE TITLE
分类loss用的是pred_prob = tf.sigmoid(conv_raw_prob)

### ISSUE THREAD

**Issue by henbucuoshanghai** [klauspa/Yolov4-tensorflow#7]
为什么呢？

**Comment by henbucuoshanghai** [comment]
why the loss of classifiation is  tf.sigmoid?     why not the softmax loss??????????????
